### ISSUE TITLE
Add tolerations to prometheus k8s master DaemonSet

### ISSUE THREAD

**Issue by stephen-soltesz** [m-lab/k8s-support#100]
To monitor the k8s platform master we wish to run on prometheus instance on each k8s master VM. To allow the service to run on nodes under many circumstances without being evicted, the daemonset should include "tolerations" of known or likely conditions on the master nodes.

Known tolerations: `NetworkUnavailable=true:NoSchedule`

Candidate tolerations: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/#taint-based-evictions

**Comment by nkinkade** [comment]
I believe this [issue is resolved](https://github.com/m-lab/k8s-support/pull/102/files#diff-a08344d02022f8bcf09544519d5ed03cR33) in PR #102.

In this PR I elected to only add a toleration for the `network-unavailable:NoSchedule` taint, since the other standard/known taints that might affect a master server are pretty severe, and I reasoned that if we have, for example, a `not-ready:NoExecute` taint that it's probably best to understand why we have that before we worry about getting Prometheus running. Similarly, if we have a `memory-pressure:NoSchedule` taint or a `unreachable:NoExecute` taint, then we are probably more worried about why these taints exists in the first place than we are in trying to force a Prometheus pod to deploy to the node, perhaps exacerbating the underlying condition.
